index.html

```diff
@@ -222,33 +222,39 @@ <h2>About me</h2>
                             </div>
                         </div>
                     </section>
-            <!-- start of contact section  -->
 
-            <!--Contact form section added by Charles Andrew-->
+
+                    <!-- start of contact section --> 
+
+                    <!--Contact form section added by Celyne Birungi-->
                     
-                    <section class="contactFormSec">
-                        <div class="body-area">
-                          <div class="contactFormSec_para">
-                            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-                          </div>
-                          
-                          <form action="https://formspree.io/f/xyyangae" method="post">
-                            <input type="text" class="text-field" placeholder="Full name" minlength="1" maxlength="30" name="Name" required>
-                            <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
-                            <textarea name="short-writeup" id="bio" class="text-field" cols="30" rows="5" minlength="1" maxlength="500" placeholder="Write me something..." required></textarea>
-                            <div class="form-button">
-                                  <button type="submit" class="about-me-btn btn-form">Get in touch</button>
+                    <section class="contactFormSec">    
+                        <div class="body-area"> 
+
+                            <div class="contactFormSec_para">   
+                                 <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>   
                             </div>
-                                
-                          </form>
-                        </div>
+                            <form action="https://formspree.io/f/xyyangae" method="post">    
+                                <input type="text" class="text-field" placeholder="Full name" minlength="1" maxlength="30" name="Name" required>
+                                <input type="email" class="text-field" placeholder="Email address" name="Email address" required>
+                                <textarea name="short-writeup" id="bio" class="text-field" cols="30" rows="5" minlength="1" maxlength="500" placeholder="Write me something..." required></textarea>
```

Comment:
- [ ] According to the requirement, it seems all the input fields should be required, and I guess the `textarea` is an input field also, if not then it'll be an output field 🤣. Just kidding!
- Kindly make it required also.🙂

Reply:
It was already there.